data/data.js

```diff
@@ -1,73 +1,73 @@
-    "ml": ["1973", "ML言語は、1973年にRobin Milnerによって開発されたプログラミング言語である。", "ML"],
-    "moon": ["2011", "MoonScriptは、2011年にLeafoによって開発されたプログラミング言語である。", "MoonScript"],
-    "nako": ["2011", "Nadesikoは、2011年にHatenaによって開発されたプログラミング言語である。", "Nadesiko"],
-    "nim": ["2008", "Nimは、2008年にAndreas Rumpfによって開発されたプログラミング言語である。", "Nim"],
-    "nscripter": ["1998", "NScripterは、1998年にNaoki Takahashiによって開発されたプログラミング言語である。", "NScripter"],
-    "p": ["1970", "Pascalは、1970年にNiklaus Wirthによって開発されたプログラミング言語である。", "Pascal"],
-    "pde": ["2001", "PDEは、2001年にProcessing Foundationによって開発されたプログラミング言語である。", "PDE"],
-    "php": ["1995", "PHPは、1995年にRasmus Lerdorfによって開発されたプログラミング言語である。", "PHP"],
-    "pl": ["1964", "PL/Iは、1964年にIBMによって開発されたプログラミング言語である。", "PL/I"],
-    "ps1": ["2006", "Powershellは、2006年にMicrosoftによって開発されたプログラミング言語である。", "Powershell"],
-    "py": ["1991", "Pythonは、1991年にGuido van Rossumによって開発されたプログラミング言語である。", "Python"],
-    "raku": ["2000", "Rakuは、2000年にLarry Wallによって開発されたプログラミング言語である。", "Raku"],
-    "rb": ["1995", "Rubyは、1995年にYukihiro Matsumotoによって開発されたプログラミング言語である。", "Ruby"],
-    "rdr": ["2018", "RDRは、2018年にRDR Groupによって開発されたプログラミング言語である。", "RDR"],
-    "re": ["2015", "Reasonは、2015年にFacebookによって開発されたプログラミング言語である。", "Reason"],
-    "res": ["2018", "Rescriptは、2018年にJordan Walkeによって開発されたプログラミング言語である。", "Rescript"],
-    "rack": ["1995", "Racketは、1995年にMatthew Flattによって開発されたプログラミング言語である。", "Racket"],
-    "rs": ["2010", "Rustは、2010年にMozilla Researchによって開発されたプログラミング言語である。", "Rust"],
-    "sb3": ["2003", "Scratchは、2003年にMIT Media Labによって開発されたプログラミング言語である。", "Scratch"],
-    "scala": ["2003", "Scalaは、2003年にMartin Oderskyによって開発されたプログラミング言語である。", "Scala"],
-    "scm": ["1975", "Schemeは、1975年にGerald Jay Sussmanによって開発されたプログラミング言語である。", "Scheme"],
-    "svelte": ["2016", "Svelteは、2016年にRich Harrisによって開発されたプログラミング言語である。", "Svelte"],
-    "swift": ["2014", "Swiftは、2014年にAppleによって開発されたプログラミング言語である。", "Swift"],
-    "tex": ["1978", "TeXは、1978年にDonald Knuthによって開発されたプログラミング言語である。", "TeX"],
-    "ts": ["2012", "TypeScriptは、2012年にMicrosoftによって開発されたプログラミング言語である。", "TypeScript"],
-    "v": ["2019", "Vは、2019年にAlexander Medvednikovによって開発されたプログラミング言語である。", "V"],
-    "vb": ["1996", "VBScriptは、1996年にMicrosoftによって開発されたプログラミング言語である。", "VBScript"],
-    "vim": ["1991", "Vim scriptは、1991年にBram Moolenaarによって開発されたプログラミング言語である。", "Vim script"],
-    "vue": ["2014", "Vue.jsは、2014年にEvan Youによって開発されたプログラミング言語である。", "Vue.js"],
-    "wat": ["2018", "watは、2018年にWebAssembly Community Groupによって開発されたプログラミング言語である。", "wat"],
-    "wy": ["2018", "WenyanLangは、2018年にYiyi Wangによって開発されたプログラミング言語である。", "WenyanLang"],
-    "zen": ["2015", "ZenScriptは、2015年にCraftTweakerによって開発されたプログラミング言語である。", "ZenScript"],
-    "zig": ["2015", "Zigは、2015年にAndrew Kelleyによって開発されたプログラミング言語である。", "Zig"],
+    "adb": ["2003", "Androidデバイスと通信するためのコマンドラインツール。デバッグや他の開発目的に利用される", "Android Debug Bridge"],
+    "appleScript": ["1993", " AppleのMac OSで使用される自動化スクリプト言語", "AppleScript"],
```

Comment:
参考: https://en.wikipedia.org/wiki/AppleScript

```suggestion
    "applescript": ["1993", " AppleのMac OSで使用される自動化スクリプト言語", "AppleScript"],
```